api/src/pcapi/core/mails/backends/logger.py

```diff
@@ -17,6 +17,9 @@ class LoggerBackend(BaseBackend):
     when performing load tests when we don't want to overload Sendinblue.
     """
 
+    def __init__(self, is_pro: bool) -> None:
```

Comment:
Suggestion : Il pourrait être intéressant, en environnement de développement comme pour les tests, de conserver ce `is_pro` dans l'instance afin de l'affcher dans le log « A request would be sent ».